Doc/library/importlib.rst

```diff
@@ -792,6 +792,96 @@ ABC hierarchy::
         itself does not end in ``__init__``.
 
 
+:mod:`importlib.metadata` -- Metadata
+-------------------------------------
+
+.. module:: importlib.metadata
+    :synopsis: Installed package metadata reading
+
+**Source code:** :source:`Lib/importlib/metadata.py`
+
+--------------
+
+.. versionadded:: 3.8
+
+This module leverages Python's import system to provide access to installed
+package metadata, such as the package's version, entry points, and more.
+
+.. note::
+
+   This module provides functionality similar to `pkg_resources
+   <https://setuptools.readthedocs.io/en/latest/pkg_resources.html>`_ `entry
+   point API
+   <https://setuptools.readthedocs.io/en/latest/pkg_resources.html#entry-points>`_
+   and `metadata API
+   <https://setuptools.readthedocs.io/en/latest/pkg_resources.html#metadata-api>`_.
+
+   The standalone backport of this module provides more information
```

Comment:
"stand-alone"

Reply:
We'll want to address this on the next port, as this phrase is local to the CPython port.